share/ci/pypicongpu_generator.py

```diff
@@ -127,6 +129,50 @@ def get_all_major_pypi_versions(package_name):
     return [str(v) for v in version_map.values()]
 
 
+@typeguard.typechecked
+def combine_requirements(one: pkg_resources.Requirement, two: pkg_resources.Requirement) -> pkg_resources.Requirement:
+    if one.project_name != two.project_name:
+        print(f"requirements for {one.project_name} and {two.project_name} are incomparable!")
```

Comment:
Is this print sufficient or should an exception be raised

Reply:
good catch the if is missing an `exit(1)` call afterwards